<launch>
	<node name="obstacle_detection" pkg="obstacle_removal" type="obstacle_detection_debug" >
		<param name="min_obstacle_rad" value="0.4" />
		<param name="max_obstacle_rad" value="0.6" type="double" />
		<param name="scan_threshold" value="100" type="int" />
		<param name="inflation_factor" value="1" type="double" />
		<param name="max_detect_range" value="25.0" />
		<param name="combine_scans" value="5" />
		<param name="lidar_height" value="0.5" />
		<param name="obstacle_height" value="1.0" />
	</node>
</launch>
